pelican_flickrtag/plugin.py

```diff
@@ -101,17 +119,25 @@ def replace_article_tags(generator):
             logger.info('[flickrtag]: Fetching photo information for %s' % id)
             photo = api.Photo(id=id)
             # Trigger the API call...
-            photo_mapping[id] = {
-                'title': photo.title,
-                'raw_url': url_for_alias(photo, size_alias),
-                'url': photo.url,
-            }
-
-            if include_dimensions:
-                sizes = photo.getSizes()
-                size = size_for_alias(sizes, size_alias)
-                photo_mapping[id]['width'] = size['width']
-                photo_mapping[id]['height'] = size['height']
+            try:
+                photo_mapping[id] = {
+                    'title': photo.title,
+                    'raw_url': url_for_alias(photo, size_alias),
+                    'url': photo.url,
+                }
+
+                if include_dimensions:
+                    sizes = photo.getSizes()
+                    size = size_for_alias(sizes, size_alias)
+                    photo_mapping[id]['width'] = size['width']
+                    photo_mapping[id]['height'] = size['height']
+
+            except:
```

Comment:
In a collaborative way of works, not everyone got flickr_api. when we write pages if we dont get them a place holder pict is displayed. will probably get a better way to do this.